src/field_theory/ratfunc.lean

```diff
@@ -87,6 +93,32 @@ lemma to_fraction_ring_injective :
   function.injective (to_fraction_ring : _ → fraction_ring (polynomial K))
 | ⟨x⟩ ⟨y⟩ rfl := rfl
 
+/-- Non-dependent recursion principle for `ratfunc K`: if `f p q : P` for all `p q`,
```

Comment:
Please spell out in a bit more detail what `P` and `f` are. Maybe start with something like:
```suggestion
/-- Non-dependent recursion principle for `ratfunc K`:
To construct a term of `P : Sort*` out of `x : ratfunc K`, it suffices to provide `f : Π (p q : polynomial K), P`
such that ...
  if `f p q : P` for all `p q`,
```